README.md

```diff
@@ -19,7 +19,7 @@ make clean test
 ```
 
 > Note: if you are running on Windows,
-[this article includes important notes for compiling Elixir from source
+[This article includes important notes for compiling Elixir from source
```

Comment:
Since the previous character is a comma, we should keep lowercase here.

Reply:
@josevalim  Yes correct. I will revert it.